web-marketplace/src/pages/Dashboard/hooks/useProfileItems.tsx

```diff
@@ -9,12 +11,14 @@ type Props = {
 };
 
 export const useProfileItems = ({ address, accountId }: Props) => {
+  const { activeAccountId } = useAuth();
   const { isIssuer } = useQueryIsIssuer({ address });
   const isCreditClassCreator = useQueryIfCreditClassCreator({ address });
   const { isProjectAdmin } = useQueryIsProjectAdmin({ address, accountId });
   const isCreditClassAdmin = useQueryIsClassAdmin({ address });
 
-  const showProjects = isProjectAdmin || !!accountId;
+  const activeAccountProfile = !!activeAccountId && !accountId && !address;
```

Comment:
We might want to have that directly in `useAuth` since we're likely to reuse it in different places.

Reply:
`accountId` and `address` comes from the `useProfileItems` params so not sure if that makes sense to have `!!activeAccountId` directly in `useAuth`
this is just to verify that we're looking at the current profile (/profile) with a logged in account in which case, we always want to display the "projects" tab